test/libsolidity/syntaxTests/array/length/const_cannot_be_fractional.sol

```diff
@@ -3,4 +3,4 @@ contract C {
     uint[L] ids;
 }
 // ----
-// TypeError 5462: (51-52): Invalid array length, expected integer literal or constant expression.
+// UnimplementedFeatureError: Not yet Implemented - FixedPointType.
```

Comment:
Note to self: This means, we are bailing out early (as soon as a fixed point type is analysed in source).